tools/clang/lib/Headers/hlsl/dx/linalg.h

```diff
@@ -0,0 +1,172 @@
+// Header for linear algebra APIs.
```

Comment:
Wonder if we should use ifdef's to exclude this if language version is before 2021 and SM < 6.9?